constellations/orion/src/repos/migrations/000026_create_table_user_afh.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE user_afh
+(
+    id          int unsigned NOT NULL AUTO_INCREMENT,
+    user_id     int          NOT NULL,
+    afh_id      int          NOT NULL,
+    PRIMARY KEY (id)
+) AUTO_INCREMENT = 1
+  DEFAULT CHARSET = UTF8MB4;
```

Comment:
`user_id` and `afh_id` should be foreign keys! Look at `classes` or `users` to see how to specify a foreign key. With foreign keys, they automatically ensure that "linked" entities exist.

For example, if I created a user_afh for user 432 and afh session 14, foreign keys ensure that user 432 exists in the user table and a afh session exists with id 14.